fuelup-init.sh

```diff
@@ -226,8 +226,8 @@ downloader() {
         fi
         if [ -n "$_err" ]; then
             echo "$_err" >&2
-            if echo "$_err" | grep -q 404$; then
-                err "installer for platform '$3' not found, this may be unsupported"
+            if echo "$_err" | grep -q 404; then
+		err "fuelup was not found - either the release is not ready yet or the tag is invalid. You can check if the release is available here: https://github.com/FuelLabs/fuelup/releases"
```

Comment:
Link to the actual release that's being tried

Reply:
Hmm, I linked to the releases page as a conscious decision since linking to a version that doesn't exist results simply in a Github 404 page, which isn't very informative (imo). Linking to the releases page allows the user to look at the latest release from the top anyway, and to manually check if the version the script was trying to download exists.

With that said, will update it to link to the actual release though - if the above logic makes sense I can change it back in future

Reply:
Unless something goes very wrong, the only time this error should be hit under normal operation is when there's a release but not binaries uploaded yet. So not a 404. If it's a 404 then that means something went really wrong, for which linking to the exact page is probably more informative.